Add Live from GodotCon Boston Web .NET prototype blogpost

adamscott opened the pull request:
![Screenshot 2025-05-07 at 19-28-32 Live from GodotCon Boston Web  NET prototype – Godot Engine](https://github.com/user-attachments/assets/92665f6c-f65c-4cb4-88f8-45c3b4f9175c)

adamscott committed:
Add Live from GodotCon Boston Web .NET prototype blogpost

akien-mga commented on collections/_article/live-from-godotcon-boston-web-dotnet-prototype.md:
```diff
@@ -0,0 +1,105 @@
+---
+title: "Live from GodotCon Boston: Web .NET prototype"
+excerpt: ""
```
TODO

adamscott replied:
Done!

akien-mga commented on storage/blog/live-from-godotcon-boston-web-net-prototype/slides_state-of-the-godot-and-the-web-2025/index.html:
```diff
@@ -0,0 +1,1731 @@
+<!DOCTYPE html>
```
That's a lot of code and data/fonts for a slick deck that's only linked in a blog post. Is this the framework you built your presentation with or did you convert it all to HTML just for the blog post? Because otherwise a PDF would likely do just fine.

adamscott replied:
It's actually the exported slides from [slides.com](slides.com) that uses [the open source framework reveal.js](https://revealjs.com/) internally. There is some sound effects/video that would be lost though if not included.

I thought linking directly to the public url for my slides on slides.com, but I thought that hosting the files directly would maybe prevent some link rot.

adamscott replied:
Though, I could remove unused fonts. It seems that it tries to package fonts that I didn't even use.

JohnVeness replied:
Is it possible to view this slideshow (with sound effects/video) somehow, without having to build the site locally?

adamscott replied:
Did remove unused fonts, and it work flawlessly. The number of files this PR was dramatically reduced.

I just tried to export the slides to PDF and it's all broken, unfortunately.

coppolaemilio commented:
@adamscott did @raulsntos reviewed this? I would like his approval first before publishing

adamscott commented:
> @adamscott did @raulsntos reviewed this? I would like his approval first before publishing

Yes, and I replaced some content at his demand! :)

coppolaemilio commented:
I've tested the page and it looks okay, but I'm not comfortable with us also hosting the presentation and including all those libraries. If you want to link to the slides you should host it somewhere else and link there or export a single file like a PDF that anyone can view

adamscott commented:
> I've tested the page and it looks okay, but I'm not comfortable with us also hosting the presentation and including all those libraries. If you want to link to the slides you should host it somewhere else and link there or export a single file like a PDF that anyone can view

I changed the links to point to https://slides.com/scottmada/state-of-the-godot-and-the-web-2025 instead, and removed the local hosting.

coppolaemilio committed:
replacing "leader" for "lead" again

coppolaemilio committed:
Change button style and copy

coppolaemilio commented:
Thanks!
